lecture_02/assignment_01/ana-ascic/assignment_01.py

```diff
@@ -0,0 +1,32 @@
+from compas.artists import MeshArtist
```

Comment:
I would recommend always using the generic artist:
```suggestion
from compas.artists import Artist
```